package.json

```diff
@@ -25,11 +25,8 @@
     "start": "nx start",
     "serve": "nx serve",
     "build": "nx build",
-    "build:libs": "nx run-many --target=build --parallel=false --with-deps=true --projects=remix-analyzer,remix-astwalker,remix-debug,remix-lib,remix-simulator,remix-solidity,remix-tests,remix-url-resolver",
```

Comment:
Why did you take out this build script? Will this not break ci?

Reply:
I has been moved below here: https://github.com/ethereum/remix-project/pull/260/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R47